crates/curp/src/server/curp_node.rs

```diff
@@ -522,54 +523,50 @@ impl<C: Command, RC: RoleChange> CurpNode<C, RC> {
         let batch_timeout = curp.cfg().batch_timeout;
         let leader_event = curp.leader_event();
 
-        #[allow(clippy::arithmetic_side_effects, clippy::ignored_unit_patterns)] // tokio select internal triggered
-        #[allow(clippy::never_loop)] // false positive
-        'outer: loop {
```

Comment:
Why this loop is removed?

Reply:
That‘s because the outer loop is useless, it never loops